packages/matchbox-icons/icons/ArrowForward.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { createSvgIcon } from './IconBase';
+
+export default createSvgIcon(
```

Comment:
Is it worth making this functional and using a base icon component that accepts the path as a prop?

Reply:
Hmm not sure, I followed material-ui's implementation: https://github.com/mui-org/material-ui/tree/v1-beta/packages/material-ui-icons

Only difference is `IconBase` is in this package instead of `matchbox`. I don't see any reason why they couldn't be functional components, just not sure how to easily edit 900+ files to make this change.